Gemfile

```
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+ruby RUBY_VERSION
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+gem "github-pages", group: :jekyll_plugins
```

```suggestion
gem "github-pages", group: :jekyll_plugins
gem "webrick", "~> 1.7"
```

Required if using Ruby 3+, as far as I can tell.